docs/zh/latest/plugins/batch-requests.md

```diff
@@ -52,8 +52,7 @@ title: batch-requests
 
 ## 接口
 
-插件会增加 `/apisix/batch-requests` 这个接口，你可能需要通过 [interceptors](../plugin-interceptors.md)
-来保护它。
+插件会增加 `/apisix/batch-requests` 这个接口，你可能需要通过 [public-api](public-api.md) 插件来暴露它。
```

Comment:
```suggestion
插件会增加 `/apisix/batch-requests` 这个接口，你需要通过 [public-api](public-api.md) 插件来暴露它。
```

Reply:
fixed

Reply:
updated, please re-check and resolve it. XD